44.md

```diff
@@ -1,74 +1,150 @@
-# NIP-44
+NIP-44
+=====
 
-## Encrypted Payloads (Versioned)
+Encrypted Payloads (Versioned)
+------------------------------
 
 `optional`
 
 The NIP introduces a new data format for keypair-based encryption. This NIP is versioned
-to allow multiple algorithm choices to exist simultaneously.
+to allow multiple algorithm choices to exist simultaneously. This format may be used for
+many things, but MUST be used in the context of a signed event as described in NIP 01.
 
-Nostr is a key directory. Every nostr user has their own public key, which solves key
-distribution problems present in other solutions. The goal of this NIP is to have a
-simple way to send messages between nostr accounts that cannot be read by everyone.
+*Note*: this format DOES NOT define any `kind`s related to a new direct messaging standard,
```

Comment:
@staab apologies for commenting on a very old PR like this, but I'm curious if you could provide guidance on *why* NIP-44 shouldn't be used as a drop-in replacement for NIP-04.

I'm investigating the best way to replace the NIP-04 usage in NIP-47 (NWC), and I would love to understand the tradeoffs here (should messages be gift wrapped, etc).

Reply:
There is ambiguity between NIP-04, the encryption scheme, and NIP-04, the DM kind. NIP-44 is a replacement for the encryption scheme, not the DM scheme. 

It should not be replaced in `kind:4` DMs because of all the metadata leaks that are incompatible with the DM use case. On NIP-47, you can just use NIP-44 like you have been using NIP-04 encryptions.

Reply:
Right, nip-04-type dms with nip44 encryption would also be incompatible with all other messaging specs. Using nip 44's encryption for NWC/NC is encouraged, with the caveat that decryption support should happen before people start encrypting messages with nip 44 to avoid interoperability problems.

Reply:
That makes sense, thank you! I'm also thinking through how a migration path might work here.